CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.0.1] - 2019-01-16
```

Comment:
Please change to 1.1.0 as this will be a feature release and not a patchlevel relese